### DISCUSSION TITLE
Matter Commissioning successful - on iPhone HomeApp: Matter Accessery : No Response

### DISCUSSION THREAD

**Discussion by alexwala** [arendst/Tasmota#21933] [Q&A] [answered]
hi !

using tasmota version 14.1.0 (TasmoCompiler-esp32s3) with Matter enabled (LILYGO T-Dongle-S3)

after adding tasmota accessory to HomeKit got Commissioning complete:

```
--- Commissioning complete for Fabric '00000000xxxxxxxx' (Vendor Apple Home) ---
--- Commissioning complete for Fabric '00000000xxxxxxxx' (Vendor Apple Keychain) ---
```

but after all the tasmota accessory shows a 'No Response' in HomeApp

Environment
- iPad as HomeHub
- eero router

did I forgot something ?

thx alex

**Comment by Jason2866** [comment, marked as answer]
You can't use a iPad as Hub. It needs at least a HomePod mini or an Apple TV 4K (3. Generation)

**Reply by alexwala** [reply]
ok thanks - so only the commissioning works fine with an iPad as Hub...but after that matter isn't working - correct ?

**Reply by s-hadinger** [reply]
Apple dropped iPad as home hub. It's an anomaly that the commissioning worked, it shouldn't

**Reply by alexwala** [reply]
Yep commissionig worked fine....
But now i tried with a Google Nest Mini 2 but unable to pair with the tasmota...(via iPhone Google Home App)

**Reply by s-hadinger** [reply]
I have a nest mini 2 and iphone and it pairs fine. 

Please paste logs with `weblog 3` level

**Reply by alexwala** [reply]
Interesting - 
On "Next, your device will be added to iOS and Google Home" i always get an 'Unable to Add Accessory' error
But why i will be added to iOS too ?
ok so with tasmota version 14.1.0 it should work ...

**Reply by s-hadinger** [reply]
Logs....

**Reply by alexwala** [reply]
```
15:11:22.591` MTR: New Commissioning (PASE id=35804) from [810b:0f08:4fda:c82d:0ad3%st1]:64066
15:11:27.413 RSL: RESULT = {"Matter":{"Commissioning":0}}
15:11:28.941 MTR: New Connection (CASE id=17906) from [810b:0f08:4fda:c82d:0ad3%st1]:64066
15:11:29.245 MTR: =Saved     0 fabric(s) and 0 session(s)
15:11:29.464 MTR: =Saved     1 fabric(s) and 1 session(s)
15:11:29.519 MTR: --- Commissioning complete for Fabric '00000000xxxxxxxx' (Vendor Apple Keychain) ---
15:11:34.495 MTR: New Commissioning (PASE id=58658) from [810b:0f08:4fda:c82d:0ad3%st1]:53265
```

**Reply by s-hadinger** [reply]
As said earlier, please set log level to 3. Use `weblog 3` for the web UI or `seriallog 3` for serial port

**Reply by alexwala** [reply]
I'm currently busy but i will re-do the pairing with `weblog 3` / `seriallog 3` and paste the log's - thanks in advance !

**Reply by s-hadinger** [reply]
The first phase of pairing is done from the iPhone. Later it is done from the nest mini. One possibility is that your nest mini cannot reach the Tasmota device. Are they on the same vlan?

**Reply by alexwala** [reply]
jep they are on the same wlan, but see my [comment ](https://github.com/arendst/Tasmota/discussions/21933#discussioncomment-10282500)😉

**Comment by alexwala** [comment]
had success yesterday with an android phone and the nest mini:

1. added the tasmota as matter accessory on the android phone
2. on the iphone's google app i accepted an invitation request to the 'home' created on the android phone
3. now i can control the tasmota device via iphone's google home app 😉

the correct required Home Hub (currently iPad as Home Hub) may be the reason why pairing does not work directly via the phone's Google Home app: it tries to register to ios and google home
